### PR TITLE
MSYS2 change MINGW64 to UCRT64

### PR DESCRIPTION
**Pull request by AnHeuermann**
### Related Issues

For OpenModelica we want to use the newer Universal C Runtime (UCRT) on Windows, see https://github.com/OpenModelica/OpenModelica/pull/10939.
For this it would also be good if OMSimulator can be build with MSYS2 and UCRT64 as well as MINGW64.

## Changes
  - Adding `OMDEV_MSYS` environment variables to Jenkinsfile and build scripts.
  - Remove 32 bit support (there is no 32 bit Windows, Ubuntu any more)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -345,6 +345,25 @@ EXIT /b 1
...
+            label 'omsimulator-windows'
+          }
+          environment {
+            OMDEV = "C:\\OMDev"
+            OMDEV_MSYS = "${env.OMDEV}\\tools\\msys"
+            MSYSTEM_DIR_PREFIX = "ucrt64"
+            PATH = "${env.PATH};${env.OMDEV_MSYS}\\${env.MSYSTEM_DIR_PREFIX}\\bin\\;${env.OMDEV_MSYS}\\usr\\bin;C:\\bin\\git\\bin;C:\\bin\\git\\usr\\bin;"
+            CC="clang"
+            CXX="clang++"
+            MSYSTEM = "MINGW64"
```
Comment (AnHeuermann) [inline Jenkinsfile:359]: This should be `MSYSTEM = "UCRT64"`.

**Comment by arun3688** [comment]
> I think all these were done in #1286, please check.

yes @AnHeuermann I hope we can close this PR

**Comment by lochel** [comment]
@AnHeuermann, @adrpo, @arun3688 Could you provide an update on the status of this pull request?

**Comment by lochel** [comment]
This is implemented by other pull requests and can be closed according to @arun3688.

* #1286

**Pull request closed without merging**